### PR TITLE
navitia: A new entity durations with two attributs(total and walking) added in journey

### PR DESCRIPTION
**Pull request by kadhikari**
Tickets concerned:
http://jira.canaltp.fr/browse/ITI-9
http://jira.canaltp.fr/browse/ITI-10

### CODE REVIEW

**Comment by TeXitoi** [comment]
Jenkins, test this please

**Comment by kadhikari** [comment]
Jenkins, test this please

File: source/routing/raptor_api.cpp [hunk 1]
```diff
@@ -206,6 +206,22 @@ static void co2_emission_aggregator(pbnavitia::Journey* pb_journey){
     }
 }
 
+static void calculate_total_walking_duration(pbnavitia::Journey* pb_journey) {
```
Comment (TeXitoi) [inline source/routing/raptor_api.cpp]: Please rename it `compute_durations` as the fields in the durations object may change later.

**Comment by TeXitoi** [comment]
Jenkins, test this please

**Comment by kadhikari** [comment]
@TeXitoi I renamed the function as you mentioned

**Comment by TeXitoi** [comment]
OK for me.

**Pull request merged**